src/bin/cql-stress-cassandra-stress/settings/command/user.rs

```diff
@@ -100,7 +102,10 @@ pub struct UserParams {
     pub keyspace_definition: Option<String>,
     pub table: String,
     pub table_definition: Option<String>,
-    pub queries: HashMap<String, QueryDefinition>,
+    // Maps a query name to query definition and a ratio with which
+    // this query will be sampled.
+    pub queries_payload: HashMap<String, (QueryDefinition, f64)>,
```

Comment:
WDYT about introducing a type alias `type OpWeight = f64` ? Would it impact readability positively or negatively?